### PR TITLE
Migrate remaining C strings in config handling

### PR DESCRIPTION
**Pull request by Gymnasiast**

### CODE REVIEW

File: src/openrct2/config/Config.cpp [hunk 1]
```diff
@@ -511,8 +511,8 @@ namespace Config
         if (reader->ReadSection("font"))
         {
             auto model = &gConfigFonts;
-            model->FileName = reader->GetCString("file_name", nullptr);
-            model->FontName = reader->GetCString("font_name", nullptr);
+            model->FileName = reader->GetString("file_name", nullptr);
```
Comment (frutiemax) [inline src/openrct2/config/Config.cpp:514]: Put nullptr to ""

**Review by frutiemax — COMMENTED** [review]
Put "" instead of nullptr of std::string

File: src/openrct2/config/Config.cpp (hunk 1 above)
Comment (Gymnasiast) [reply src/openrct2/config/Config.cpp:514]: Damn, I was so focused on the ones in IniReaderTest that I completely overlooked these :/

**Comment by Gymnasiast** [comment]
Thanks, @frutiemax , I was pulling my hair out because of that segfault!

**Review by IntelOrca — APPROVED** [review]

**Pull request merged**